### PR TITLE
Story/#mov 151 data capturing

### PR DESCRIPTION
**Pull request by muthenberg**

### CODE REVIEW

File: datacapturing/src/androidTest/java/de/cyface/datacapturing/backend/CapturedDataWriterTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ *
+ * (!) Create a custom Instrumented Test config for integration-test module to run this in the IDE !
+ *
+ * @author Klemens Muthmann
+ * @version 1.0.0
+ * @since 1.0.0
+ */
+@RunWith(AndroidJUnit4.class)
+public class CapturedDataWriterTest /*extends ProviderTestCase2<MeasuringPointsContentProvider>*/ {
+    /**
```
Comment (hb0) [inline datacapturing/src/androidTest/java/de/cyface/datacapturing/backend/CapturedDataWriterTest.java]: dead code?

File: datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java [hunk 2]
```diff
@@ -1,85 +1,146 @@
...
  * service as often as you like and reuse the object.
- * </p>
  *
  * @author Klemens Muthmann
  * @version 1.0.0
  * @since 1.0.0
  */
 public final class DataCapturingService {
 
+    /*
```
Comment (hb0) [inline datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java]: left-over comment or work-in-progress?

File: datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java [hunk 3]
```diff
@@ -1,85 +1,146 @@
...
-     * </p>
+     * <p>
+     * Since the method is synchronized with the actual service startup it should be considered a long running operation
+     * and not run on the UI thread to avoid ANR errors.
      */
     public void start() {
-        isRunning = true;
-        unsyncedMeasurements.add(new Measurement(unsyncedMeasurements.size()));
+        // TODO this will cause an error in the handler which currently does not accept null.
+        start(null);
```
Comment (hb0) [inline datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java]: wip?

Comment (muthenberg) [reply datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java]: Jep. Already fixed in the most recent version.

File: datacapturing/src/androidTest/java/de/cyface/datacapturing/backend/CapturedDataWriterTest.java (hunk 1 above)
Comment (muthenberg) [reply datacapturing/src/androidTest/java/de/cyface/datacapturing/backend/CapturedDataWriterTest.java]: Not really. I added the relevant code in a later commit under a different task.

**Pull request merged**